e2e/cypress/support/okta_commands.js

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {_} from 'lodash';
+import * as TIMEOUTS from '../fixtures/timeouts';
+
+Cypress.Commands.add('oktaGetApp', () => {
+    return cy.request({
+        method: 'GET',
+        url: Cypress.env('okta_api_url') + '/apps/' + Cypress.env('okta_mm_app_instance_id'),
+        headers: {
+            Authorization: 'SSWS ' + Cypress.env('okta_mm_app_token'),
+        }
+    }).then((response) => {
+        expect(response.status).to.be.equal(200);
+        //console.log('oktaGetApp ' + JSON.stringify(response.body));
+        return cy.wrap(response.body);
+    });
+});
+
+
+Cypress.Commands.add('oktaUpdateAppSettings', (appSettingsJson) => {
+    return cy.request({
+        method: 'PUT',
+        url: Cypress.env('okta_api_url') + '/apps/' + Cypress.env('okta_mm_app_instance_id'),
+        headers: {
+            Authorization: 'SSWS ' + Cypress.env('okta_mm_app_token'),
+        },
+        body: appSettingsJson
+    }).then((response) => {
+        expect(response.status).to.equal(200);
+        //console.log('oktaUpdateAppSettings ' + JSON.stringify(response.body));
+        return cy.wrap(response.body);
+    });
+});
+
+function buildProfile(user) {
+    const profile = {
+        firstName: user.firstname === null ? null : user.firstname,
```

Comment:
I'm confused here.  Is it not simply the same as,
```suggestion
        firstName: user.firstname,
```

Reply:
yes, it is = )